net-proxy/trojan/metadata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>yangmame@icloud.com</email>
+		<name>YangMame</name>
```

Comment:
Per the new policy, we need your real name at least on commit signoffs, so you may want to put it here as well. Technically we could merge this PR without it but then you wouldn't be able to update it.

Reply:
they are the same